### ISSUE TITLE
[JAWS + IE11] Reset all alert is heard when the dialog is opened

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#244]
Seen on Windows 10, IE11 + JAWS. For phetsims/tasks#831. Found by @phet-steele.

When a dialog is opened after pressing the reset all button, JAWS will announce the "Reset All" alert again.

When the dialog is opened, all aria-live elements have their content cleared (so they cannot be found with the virtual cursor).

Thoughts:
 - Perhaps JAWS adds "removals" as its default behavior for the aria-live sibling attribute aria-relevant? (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-relevant_attribute). Should we explicitly set "additions text" for the aria-live elements?

 - Instead of removing alert content can we hide it another way?

**Comment by jessegreenberg** [comment]
Yes, for role="alert", the JAWS treats the like aria-relevant="all" (though ONLY in IE11).  See
https://jsfiddle.net/bxho5bvu/2/

Removing role="'alert" from the aria-live element stops the alert from being read when the content is removed.

**Comment by jessegreenberg** [comment]
When I add "aria-relevant=additions" JAWS reads the alert when the content is removed.

**Comment by jessegreenberg** [comment]
Using an aria-live element without role=alert fixes the problem.

**Comment by jessegreenberg** [comment]
Replaced above, closing.

**Comment by jessegreenberg** [comment]
Reopening.  This change added a VO bug where pressing the reset all button causes focus to move to the alert elements when charges are added to the body.  Switching to aria-live="polite" fixes the problem.  Based on this, the only aria-live attribute that works with consistency is aria-live="polite".

**Comment by jessegreenberg** [comment]
Fixed, in a scenery-phet commit, tested with VO/Safari, JAWS/IE11, NVDA/Firefox
